src/DisplayWindow.cpp

```diff
@@ -193,5 +200,5 @@ void DisplayWindow::readScreen2(void * _dest, int * _width, int * _height, int _
 
 u32 DisplayWindow::maxMSAALevel() const
 {
-	return _maxMSAALevel();
+	return m_maxMsaa;
```

Comment:
WindowsWGL::maxMSAALevel() becomes useless and should be removed.

Reply:
fixed